orchestrator_mapreduce.py

```
@@ -0,0 +1,380 @@
+        from protocols.loader import load_protocol
+
+        protocol = load_protocol(protocol_name)
+        if not protocol:
+            raise Exception(f"Protocol '{protocol_name}' not found")
+
+        # Run in executor to avoid blocking the event loop
+        loop = asyncio.get_event_loop()
+        result = await loop.run_in_executor(None, protocol["task"])
+        return result
+
+    async def _retry_subtask(self, subtask: SubTask):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `_retry_subtask`

Add return type annotation: `None`

(ANN202)